WORKSPACE

```diff
@@ -4,7 +4,7 @@ load("//:version.bzl", "check_version")
 
 # Check that the user has a version between our minimum supported version of
 # Bazel and our maximum supported version of Bazel.
-check_version("0.13.0", "0.13.1")
+check_version("0.13.0", "0.14.0")
```

Comment:
I don't think this change is safe. We are strictly _not_ yet compatible with 0.14.x yet. :(